bot.js

```diff
@@ -82,7 +85,7 @@ function reply(message) {
 		var match = matchPattern(rules[i].pattern, words, 0, 0)
 		if (match) {
 			// Reconstruct a reply - if a token is a number, find the
-			// matched word from 'match', otherwise just append the word
+			// matched word from 'match', otherwordIndexse just append the word
```

Comment:
I guess this is search & replace error :-) This is why good editors have a refactoring tool that actually understand your code and don't replace just strings!